near/src/config.rs

```diff
@@ -66,6 +67,8 @@ pub const GENESIS_CONFIG_FILENAME: &str = "genesis.json";
 pub const NODE_KEY_FILE: &str = "node_key.json";
 pub const VALIDATOR_KEY_FILE: &str = "validator_key.json";
 
+const DEFAULT_TELEMETRY_URL: &str = "https://explorer.nearprotocol.com/api/nodes/";
```

Comment:
Please, remove the trailing `/` from the URL (`https://explorer.nearprotocol.com/api/nodes`). Next.js handles only the URLs without the trailing `/`. :man_shrugging: